tests/integration_test/service_control_failed_request_report_test.go

```diff
@@ -61,7 +61,7 @@ func TestServiceControlFailedRequestReport(t *testing.T) {
 			clientProtocol: "http",
 			httpMethod:     "GET",
 			method:         "/noexistoperation?key=api-key",
-			httpCallError:  "404 Not Found, {\"code\":404,\"message\":\"Request `GET /noexistoperation?key=api-key` is not defined by this API.\"}",
+			httpCallError:  "404 Not Found, {\"code\":404,\"message\":\"\"}",
```

Comment:
@qiwzhang But now that we removed the catch-all route, we don't provide the correct error message in the response body. I assume this is important because we explicitly cared about this error message in b/169095072.

I would prefer to keep the catch-all route to provide a clearer error message. It will also make the trace spans have a nice annotation.

Reply:
+1